docs/getting_started/basics/authoring.md

```diff
@@ -30,19 +37,31 @@ Certain nodes in Shift allow the creation of new plugs. To create a new plug, `R
     * *Out*: The plug will be created at the right side of the node, which corresponds to the node's outputs. This plug will have the data processed by the node.
     * *Through*: Two plugs with the same name will be created, one to the left side of the node as an input and another one to the right side of the node as an output.
 
-![Add Plug Dialog 1](../../images/create_plug.gif)
+<figure>
+      <img src="images/create_plug.gif" alt="Create Plug">
+      <figcaption><b>Figure 3</b>: Creating a new float plug.</figcaption>
+</figure>
+
 
 ### Creating New Plugs through a Connection
 
 Adding a new plug to a node can also be done while creating a connection. To achieve this, **drag and drop** a connection over the node where the new connection is to be added. This action will automatically prompt and populate the "*Add Plug*" dialog with the name, type, values, and direction information of the plug from which the connection was started from. Once the new plug is created, a connection between this plug and the one that was dragged will be established.
 
-![Add Plug Dialog 2](../../images/create_plug_drag_connection.gif)
+<figure>
+      <img src="images/create_plug_drag_connection.gif" alt="Create a plug drag and drop.">
+      <figcaption><b>Figure 4</b>: Creating a new plug by dragging and dropping.</figcaption>
+</figure>
+
 
 ## Removing a Plug
 
 To remove a plug, `Right-Click` on the plug and select the "*Remove Plug*" option. This action will delete the plug and destroy all existing connections associated with it.
 
-![Removing Plug](../../images/remove_plugs.gif)
+<figure>
+      <img src="images/remove_plugs.gif" alt="Remove Plugs">
+      <figcaption><b>Figure 5</b>: Removing a connected plug from node.</figcaption>
+</figure>
+
 
 ## Removing a Connection
```

Comment:
Should we add a gif for removing a connection to be aligned with the previous sections?

Reply:
I thought that removing a connection is such an easy thing that there's no need to add a gif